app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -35,6 +35,14 @@ def self.default_blacklisted_event_names
     )
   end
 
+  def self.without_datastores
```

Comment:
This name is misleading.  The EMS may still have other datastores (Data Domain or Export Domain).  Maybe `without_iso_datastores` would be better?